libs-scala/resources-grpc/src/main/scala/com/daml/resources/grpc/ManagedChannelResourceOwner.scala

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.resources.grpc
+
+import com.daml.resources.{AbstractResourceOwner, HasExecutionContext, Resource}
+import io.grpc.{ManagedChannel, ManagedChannelBuilder}
+
+import scala.concurrent.Future
+import scala.concurrent.duration.FiniteDuration
+
+private[grpc] final class ManagedChannelResourceOwner[Context: HasExecutionContext](
```

Comment:
Please make `com.daml.platform.apiserver.services.GrpcClientResource` use this.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/8604/commits/c584554fcf71a237bc9da7b6c5faa8019537981a